portality/migrate/seal/seal.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I just ctrl+F-ed for "migrate" in the PR's code, so apologies if I've missed it. Looking for "seal" is not a very easy endeavour in this set of code for obv reasons :).